.buildkite/src/Lib/Map.dhall

```diff
@@ -0,0 +1,4 @@
+-- Maps are lists of special "entry" records. This is made explicit here
+let Entry = { Type =  \(value: Type) -> { mapKey : Text, mapValue : value } } in
+{ Entry = Entry, Type = \(value : Type) -> List (Entry.Type value) }
+
```

Comment:
Prelude provides this functionality in it's map related exports. See `Prelude.Map.Type` and `Prelude.Map.Entry`.

Reply:
wow how did I miss that